### ISSUE TITLE
trying to share a file that is not uploaded yet, will return no result instead of showing a message

### ISSUE THREAD

**Issue by Flexmaen** [owncloud/client#6354]
### Expected behaviour
If I want to share a file that has not been fully synced, I should get an error message (unless it is possible to provide the link for the file anyway).

### Actual behaviour
Nothing happens.

### Steps to reproduce
1.  Put a bigger file in your directory that is synced via Owncloud
2.  Right click on the file in Windows Explorer and select "Owncloud => Share..."
3.  Nothing happens if the file is not fully uploaded to the cloud.

### Server configuration
Operating system:

Web server: 

Database:

PHP version:

ownCloud version:

Storage backend (external storage):

### Client configuration
Client version: Version 2.4.0 (build 8894)

Operating system: Windows 10

OS language: German (Austria)

Qt version used by client package (Linux only, see also Settings dialog):

Client package (From ownCloud or distro) (Linux only):

Installation path of client:

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

```Template for output < 10 lines```

1. Client logfile: Output of `owncloud --logwindow` or `owncloud --logfile log.txt`
(On Windows using `cmd.exe`, you might need to first `cd` into the ownCloud directory)
(See also http://doc.owncloud.org/desktop/2.2/troubleshooting.html#client-logfile )

2. Web server error log:

3. Server logfile: ownCloud log (data/owncloud.log):

**Comment by SamuAlfageme** [comment]
Hey, @Flexmaen thanks for reporting! This is actually a duplicate of https://github.com/owncloud/client/issues/4608 and has quite recently been fixed by https://github.com/owncloud/client/pull/6330 - now the entry will be disabled until the file is available to be shared on the server. 

Care to give 2.5 a spin? You can find daily versions on http://download.owncloud.com/desktop/daily/ and also if you don't want to mess with your current stable installation, you can install our "testpilot" version that can run in parallel.
